lms/djangoapps/branding/api.py

```diff
@@ -581,10 +582,12 @@ def get_logo_url(is_secure=True):
     # otherwise, use the legacy means to configure this
     university = configuration_helpers.get_value('university')
 
+    space = 'ilmx' if request is None else get_active_partner_space(request)
```

Comment:
we should define the default space name in a const variable to avoid typos/case-sensitive

Reply:
will update it on Lilac branch